opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/DefaultProvisionService.java

```diff
@@ -921,7 +920,7 @@ private NodeScanSchedule createScheduleForNode(final OnmsNode node, final boolea
 
             final Duration scanInterval = fs.getScanInterval();
 
-            if (scanInterval.getMillis() <= 0) {
+            if (scanInterval.getMillis() < 0) {
                 LOG.debug("Node ({}/{}/{}) scan interval is zero, skipping schedule.", node.getId(), node.getForeignSource(), node.getForeignId());
```

Comment:
the log should be updated too... maybe say "is disabled" instead of "is zero" or something?

Reply:
done.